src/services/classification/index.ts

```diff
@@ -0,0 +1,36 @@
+import { axiosInstance } from '@/services/client/api-client'
+
+const CERTAINTY_THRESHOLD = 0.41
+const MAIN_SLUG_REGEX = /\/terms\/categories\/([a-z0-9\-]+)/
+const SUB_SLUG_REGEX =
+  /\/terms\/categories\/[a-z0-9\-]+\/sub_categories\/([a-z0-9\-]+)/
+
+export const getCategoryForDescription = async (description: string) => {
+  const axios = axiosInstance(process.env.NEXT_PUBLIC_BASE_URL_API)
+
+  let prediction = {
+    main: 'overig',
+    sub: 'overig',
+  }
+
+  try {
+    const response = await axios.post(`/signals/category/prediction`, {
```

Comment:
Misschien ook even kijken of de response ok is?

Reply:
Als de response niet oke is wordt dit als goed is afgevangen door de catch()

Reply:
De opmerking van Yuri klopt wel. De catch vangt af als een request faalt door een netwerkprobleem. Maar als je bijvoorbeeld een status 404 terugkrijgt, dan krijg je geen exceptie, maar is het request.ok wel false.